docs/digger/installation/macos-prereqs.asciidoc

```diff
@@ -0,0 +1,31 @@
+=== Prerequisites
+==== Machine prerequisites
+
+* A user with sudo permissions and SSH access.
```

Comment:
perhaps link to the createuser.sh file here to show user how to use it? Or a few steps here to describe how the user account should be created.